Sources/Profile-Modify.php

```diff
@@ -1342,13 +1342,14 @@ function makeCustomFieldChanges($memID, $area, $sanitize = true, $returnErrors =
 				array('id_theme', 'variable', 'id_member')
 			);
 		if (!empty($deletes))
-			$smcFunc['db_query']('', '
-				DELETE FROM {db_prefix}themes
-				WHERE id_theme = {int:id_theme} AND
-					variable = {string:variable} AND
-					id_member = {int:id_member}',
-				$deletes
-			);
+			foreach($deletes as $delete)
+				$smcFunc['db_query']('', '
+					DELETE FROM {db_prefix}themes
+					WHERE id_theme = {int:id_theme} AND
+						variable = {string:variable} AND
```

Comment:
Nit: SMF's coding style prefers the `AND` to be placed at the start of the next line rather than at the end of the previous line.